nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -1320,7 +1323,23 @@ public ImmutableSetMultimap<MethodRef, Integer> failIfNullParameters() {
 
     @Override
     public ImmutableSetMultimap<MethodRef, Integer> explicitlyNullableParameters() {
-      return ImmutableSetMultimap.of();
+      ImmutableSetMultimap.Builder<MethodRef, Integer> mapBuilder =
+          new ImmutableSetMultimap.Builder<>();
+      for (Map.Entry<String, Map<String, Map<Integer, Set<String>>>> outerEntry :
+          argAnnotCache.entrySet()) {
+        String className = outerEntry.getKey();
+        for (Map.Entry<String, Map<Integer, Set<String>>> innerEntry :
+            outerEntry.getValue().entrySet()) {
+          String methodName = innerEntry.getKey().substring(innerEntry.getKey().indexOf(" ") + 1);
+          for (Map.Entry<Integer, Set<String>> entry : innerEntry.getValue().entrySet()) {
+            Integer index = entry.getKey();
+            if (index >= 0) {
+              mapBuilder.put(methodRef(className, methodName), index);
+            }
```

Comment:
Why do we need to check if `index >= 0`?  And don't we need to check if the annotation is `@Nullable`?  This seems to add something to the map for any annotation.  If we can, we should add a test so that if the annotation is `@NonNull` we skip it (not sure if this is possible).

Reply:
I added the `index>=0` check because `argAnnotCache` holds the annotation for the return of the method with the index `-1` and for all the parameter annotations with their respective indices. I will add the `@Nullable` check and the corresponding test case.